test/fixtures/ast/mustache-errors-in-textarea/source.vue

```diff
@@ -0,0 +1,3 @@
+<template>
+    <textarea>{{foo bar}}</textarea>
```

Comment:
Is there any purpose to using an invalid expression?
If we just want to validate the token type, I think it would be fine to use code similar to the example in #261.

```suggestion
    <textarea>foo{{bar}}</textarea>
```

Reply:
The invalid expression is needed because valid ones get converted to JS tokens like `Identifier` and we cannot distinguish between `HTMLRCDataText` and `HTMLRawText`.

Note that the example in #261 also contains an invalid expression because of the semicolon in `bar;`.

Reply:
Thank you for the explanation :heart: